v2/internal/reconcilers/arm/azure_generic_arm_reconciler_instance.go

```diff
@@ -65,10 +65,10 @@ func (r *azureDeploymentReconcilerInstance) CreateOrUpdate(ctx context.Context)
 	r.Log.V(Verbose).Info("Reconciling resource", "action", action)
 
 	result, err := actionFunc(ctx)
+
```

Comment:
We usually have the error check (`if err != nil`) on the immediately following line (no gap) because it's a part of checking the statement worked.